src/test/resources/features/highlighting/inspections/duplicated_args_inspection.feature

```diff
@@ -32,15 +37,16 @@ Feature: Inspection for duplicated arguments in same section
     And SmkSectionDuplicatedArgsInspection inspection is enabled
     Then I expect inspection warning on <"target_2"> in <"target_2" # duplicate> with message
     """
-    This argument has already been added to '<section>' section.
+    This argument has been already added to '<section>' section.
     """
     When I check highlighting warnings
-    Then I invoke quick fix Remove duplicated element and see text:
+    Then I invoke quick fix Remove duplicated argument and see text:
     """
     <rule_like> NAME:
         <section>: "target_1", "target_2"  # duplicate
     """
     Examples:
       | section              | rule_like  |
-      | input                | checkpoint |
-      | input                | rule       |
+      | configfile           | subworkflow |
+      | input                | checkpoint  |
```

Comment:
Please add a test, that inspection isn't triggered for duplicated keyword arguments, e.g.
```
input:
  in1 = "foo",
  in1 = "foo"
```
we have a separate inspection for such case

Reply:
Done in d3530b55a64b2c25d87b8154f14ca225a634a17a